insights/tests/client/apps/test_compliance.py

```diff
@@ -305,6 +305,39 @@ def test_find_scap_policy(config, call):
     assert compliance_client.find_scap_policy('ref_id') == PATH
 
 
+@patch("insights.client.config.InsightsConfig")
+def test_find_scap_policy_with_one_datastream_file(config, tmpdir):
+    compliance_client = ComplianceClient(config)
+    dir1 = tmpdir.mkdir('scap')
+    file = dir1.join('test_file.xml')
+    file.write("""
+    <xccdf-1.2:Profile id="xccdf_org.ssgproject.content_profile_anssi_bp28_high">
+    </xccdf-1.2:Profile>
+        """)
```

Comment:
I don't really like the flie creation on the fly, but as far as I see there are no other file-based fixtures in the codebase :disappointed:

Reply:
Right, I saw a few other tests doing something similar and didn't see file-based fixtures either so I figured this was the preferred way to create a file in here